src/main/java/com/github/krzysiekjodlowski/ox/Move.java

```diff
@@ -0,0 +1,45 @@
+package com.github.krzysiekjodlowski.ox;
+
+import com.github.krzysiekjodlowski.ox.model.FieldNumber;
+import com.github.krzysiekjodlowski.ox.model.Symbol;
+
+/**
+ * Player move representation as Event.
+ *
+ * @author Krzysztof Jodlowski
+ */
+public class Move implements Event<Move> {
+  private final FieldNumber fieldNumber;
+  private final Symbol playersSymbol;
```

Comment:
Do you need symbol? BoardValidator has reference to Board object so you can get symbol calling board.getSymbol(move.fieldNumber).

Reply:
Yes, but first proper Symbol must be putted into board:)